cmd/earthly/main.go

```diff
@@ -487,9 +489,15 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 		&cli.BoolFlag{
 			Name:        "strict",
 			EnvVars:     []string{"EARTHLY_STRICT"},
-			Usage:       "Disallow usage of features that may create unreproduceable builds.",
+			Usage:       "Disallow usage of features that may create unreproduceable builds",
 			Destination: &app.strict,
 		},
+		&cli.BoolFlag{
+			Name:        "parallel-conversion",
+			EnvVars:     []string{"EARTHLY_PARALLEL_CONVERSION"},
+			Usage:       "*experimental* Enable parallel conversion, which speeds up the use of IF, WITH DOCKER --load, FROM DOCKERFILE and others",
+			Destination: &app.parallelConversion,
+		},
```

Comment:
Would maybe revise to an int flag? 0 means off, then the number signifies the total parallelism?

Reply:
:+1: Done